app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -220,24 +360,151 @@ end
         "status": 401,
         "kong_message": "No API key found in request, arr!"
     }
-    ```
+```
 
-6. Note the plugin can also be applied globally
+## More Examples
 
-   ```bash
-    $ http :8001/plugins \
-        name=exit-transformer \
-        config.handle_unknown=true \
-        config.functions=@transform.lua
+### Apply the Plugin Globally to Handle Unknown Responses
+
+Note the plugin can also be applied globally:
+
+{% navtabs %}
+{% navtab Using cURL %}
+
+```bash
+curl -X POST http://<admin-hostname>:8001/plugins/ \
+    -F "name=exit-transformer"  \
+    -F "config.handle_unknown=true" \
+    -F "config.functions=@transform.lua"
+...
+curl --header 'Host: non-existent.com' 'localhost:8000'
+```
+
+{% endnavtab %}
+{% navtab Using HTTPie %}
+
+```bash
+$ http :8001/plugins \
+    name=exit-transformer \
+    config.handle_unknown=true \
+    config.functions=@transform.lua
 
     $ http :8000 Host:non-existent.com
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Response:
+
+```bash  
     HTTP/1.1 200 OK
     ...
     X-Some-Header: some value
 
     {
         "error": true,
         "status": 404,
-        "kong_message": "no Route matched with those values, arr!"
+        "kong_message": "No Route matched with those values, arr!"
     }
-    ```
+  ```
+
+### Custom Errors by MIME Type
+
+This example shows a use case where you want custom JSON and HTML responses
+based on an [Accept header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept).
+
+Create a file named `custom-errors-by-mimetype.lua` file with the transformation
+code shown below. See the full list of HTTP response codes [above](#http-msgs).
+Include the status codes you want to customize. Any status code not listed in the
```

Comment:
```suggestion
code shown below. Based on the list of HTTP response codes [above](#http-msgs),
include the status codes you want to customize. Any status code not listed in the
```
Either this option, or flip the two sentences (Include the status codes you want to customize. See the full list...)

Reply:
Will flip because "Based on ..."  seems backwards